src/modules/transaction/components/TransactionRow/index.js

```diff
@@ -24,7 +25,8 @@ const TransactionRow = ({
     <DialogLink
```

Comment:
as it's just a link and data is a params shall we use Link insted of dialogLink ?

Reply:
I felt I should not change the original component used (but just extend it) since it would be outside the scope of this PR